### DISCUSSION TITLE
Global settings: add global display fields

### DISCUSSION THREAD

**Discussion by ku-zhen** [mdelobelle/metadatamenu#262] [Q&A]
Hello,

I konw there is a "global ignored fields" in the Global settings, but most of time, i have many fields and their values created by templates , only do i have a few fileds needed to update by metadatamenu. 

So **it's troublesome to set ignored fields one by one**，if you can add a setting that add global display fields instea of  "global ignored fields"

thanks~

**Comment by mdelobelle** [comment]
Hi you'll have to create a fileClass and define it as a "global fileClass", this way other fields (non registered in this fileClass or others) won't appear in the menus

**Comment by ku-zhen** [comment]
thanks，it useful,  but  find in that menu the date does‘t display：

![image](https://user-images.githubusercontent.com/61906420/211180259-54a1438a-c917-4343-a084-99856926cf01.png)

**Comment by mdelobelle** [comment]
I'll move this to the discussion section, can you open a separate issue for your last comment?
